src/main/java/charlotte/command/FindCommand.java

```diff
@@ -6,13 +6,29 @@
 import charlotte.task.TaskList;
 import charlotte.ui.Ui;
 
+/**
+ * Represents a command to find tasks by a given keyword.
+ */
 public class FindCommand extends Command{
     private final String keyword;
 
+    /**
+     * Constructs a FindCommand with the specified keyword.
+     *
+     * @param keyword The keyword to search for in tasks.
+     */
     public FindCommand(String keyword) {
         this.keyword = keyword;
     }
 
+    /**
```

Comment:
good use of javadoc to explain method. I had no problems understanding the code here